src/Apps/Sell/Routes/__tests__/ArtistRoute.jest.tsx

```diff
@@ -21,79 +19,74 @@ jest.mock("react-relay", () => ({
   fetchQuery: jest.fn(),
 }))
 
-const submissionMock: DeepPartial<
-  ArtistRoute_Test_Query$rawResponse["submission"]
-> = {
-  internalID: "example",
-  artist: {
-    internalID: "example-id",
-    targetSupply: {
-      isTargetSupply: true,
-    },
-    name: "Example Artist",
-  },
-}
-
 beforeEach(() => {
   mockUseRouter.mockImplementation(() => ({
     router: {
       push: mockPush,
       replace: mockReplace,
     },
+    match: { location: { pathname: "submissions/submission-id/details" } },
   }))
 })
 
 const { renderWithRelay } = setupTestWrapperTL({
   Component: (props: any) => {
-    return <ArtistRoute submission={props.submission} />
+    return (
+      <SubmissionRoute submission={props.submission}>
+        <ArtistRoute submission={props.submission} />
+      </SubmissionRoute>
+    )
```

Comment:
Updating the Artist route test to wrap the step route with `SubmissionRoute`, which includes the context provider.